strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/indexing/NugetRemoteRepositoryTest.java

```diff
@@ -0,0 +1,90 @@
+package org.carlspring.strongbox.storage.indexing;
+
+import java.io.IOException;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.inject.Inject;
+import javax.xml.bind.JAXBException;
+
+import org.carlspring.strongbox.client.ArtifactTransportException;
+import org.carlspring.strongbox.config.NugetLayoutProviderTestConfig;
+import org.carlspring.strongbox.configuration.ConfigurationManager;
+import org.carlspring.strongbox.providers.layout.LayoutProvider;
+import org.carlspring.strongbox.providers.layout.LayoutProviderRegistry;
+import org.carlspring.strongbox.repository.NugetRepositoryFeatures;
+import org.carlspring.strongbox.repository.RepositoryFeatures;
+import org.carlspring.strongbox.storage.Storage;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.testing.TestCaseWithRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * @author Sergey Bespalov
+ *
+ */
+@ContextConfiguration(classes = NugetLayoutProviderTestConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class NugetRemoteRepositoryTest
+        extends TestCaseWithRepository
+{
+
+    private static final String REPOSITORY_RELEASES = "m2pr-releases";
```

Comment:
The convention that I've used is that each test should have their own repository. The repository's name should be an abbreviation of the class name's initials. In this case -- `nrrt`.

We need to keep making sure that each of the tests is omnipotent and that none of the test depend on each other.

The same applies for the variable below.